src/routes/AboutUsPage/Tabs.jsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react';
+import './index.less';
+
+const Tabs = () => {
+  const [toggleState, setToggleState] = useState(1);
+
+  const toggleTab = (index) => {
+    setToggleState(index);
+  };
+
+  return (
+    <div className='container'>
+      <div className='bloc-tabs'>
+        <button
+          className={toggleState === 1 ? 'tabs active-tabs' : 'tabs'}
+          onClick={() => toggleTab(1)}
+        >
+          What We Do
+        </button>
+        <button
+          className={toggleState === 2 ? 'tabs active-tabs' : 'tabs'}
+          onClick={() => toggleTab(2)}
+        >
+          Our Team
+        </button>
+        <button
+          className={toggleState === 3 ? 'tabs active-tabs' : 'tabs'}
+          onClick={() => toggleTab(3)}
+        >
+          Constitution
+        </button>
+      </div>
+
+      <div className='content-tabs'>
+        <div
+          className={toggleState === 1 ? 'content  active-content' : 'content'}
+        >
+          <p>
+            UNSW QuantSoc is a new student society that aims to connect UNSW
+            students interested in mathematical finance, quantitative trading
+            and algorithmic market-making with opportunities to learn, socialise
```

Comment:
would be good to make a `Tab` component here that takes the header as a prop and then just renders the children